.gitignore

```diff
@@ -20,4 +20,6 @@
 # Go workspace file
 go.work
 
-.idea
\ No newline at end of file
+.idea
+
+go.sum
```

Comment:
I'm not sure we should include `go.sum` in `.gitignore` file.

Reply:
isn't this file generated in each dev environment? like package_lock.json for npm package manager?

Reply:
No, this isn't lock file.
https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions

Reply:
Done,  thanks for the documentation.
I deleted the invalid commit.